anker/src/instruction.rs

```diff
@@ -381,3 +399,95 @@ pub fn change_token_swap_pool(
         data: data.to_vec(),
     }
 }
+
+accounts_struct! {
+    SendRewardsAccountsMeta, SendRewardsAccountsInfo {
+        pub anker {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub solido {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub reserve_authority {
+            is_signer: false,
+            is_writable: false,
+        },
+        // Accounts for Wormhole swap.
+        pub wormhole_token_bridge_program_id {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub wormhole_core_bridge_program_id {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub payer {
+            is_signer: true,
+            is_writable: true,
+        },
+        pub config_key {
+            is_signer: false,
+            is_writable: false,
+        },
+        // From should be the UST token address.
+        pub ust_reserve_account {
+            is_signer: false,
+            is_writable: true,
+        },
+        pub ust_mint {
+            is_signer: false,
+            is_writable: true,
```

Comment:
Why does the mint need to be writable? Does the SPL token program require this for transfers? I would expect it to only be writable for mint/burn.

Reply:
It's needed for Wormhole here: https://github.com/certusone/wormhole/blob/57919f6675ed5e0123d83f6abed09f06e0390f09/solana/modules/token_bridge/program/src/instructions.rs#L371
Maybe it calls something that need this?!

Reply:
Looks like an oversight on their end, they only read from it and call `spl_token::initialize`, which doesn’t need the mint to be writable either: https://github.com/certusone/wormhole/blob/d678cb4662f5eb9b9590c0c9e9238d137b5e4226/solana/modules/token_bridge/program/src/api/transfer.rs#L142-L172

Reply:
I opened https://github.com/certusone/wormhole/issues/607 for this.